packages/page-accounts/src/modals/ProxyOverview.tsx

```diff
@@ -1,18 +1,18 @@
 // Copyright 2017-2020 @polkadot/app-staking authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import { ApiPromise } from '@polkadot/api';
 import { SubmittableExtrinsic } from '@polkadot/api/types';
 import { AccountId, ProxyDefinition, ProxyType } from '@polkadot/types/interfaces';
-
 import BN from 'bn.js';
 import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import styled from 'styled-components';
+import { ApiPromise } from '@polkadot/api';
```

Comment:
This one is perfect as an adjustment, it caught something that was incorrect. (Although I guess `ApiPromise` was only used as a `type`)